api/endpoint/basic.go

```diff
@@ -4,9 +4,15 @@ import (
 	"github.com/gin-gonic/gin"
 	"github.com/trustwallet/blockatlas/internal"
 	"github.com/trustwallet/blockatlas/pkg/blockatlas"
+	"github.com/trustwallet/blockatlas/pkg/servicerepo"
 	"net/http"
 )
 
+func Init(serviceRepo *servicerepo.ServiceRepo) {
+	initAssetsService(serviceRepo)
```

Comment:
We need to init all the services at 1 place

Reply:
done